### PR TITLE
Add various missing license headers and rename meson targets for consistency

### PR DESCRIPTION
**Pull request by keszybz**

### CODE REVIEW

File: src/test/generate-sym-test.py [hunk 1]
```diff
@@ -16,15 +16,14 @@
...
         else:
-            print('    {},'.format(s))
+            print(f'    {s},')
 
 print('''};
 
 int main(void) {
-    unsigned i;
-    for (i = 0; i < sizeof(symbols)/sizeof(void*); i++)
+    for (unsigned i = 0; i < sizeof(symbols)/sizeof(void*); i++)
```
Comment (poettering) [inline src/test/generate-sym-test.py]: size_t would be preferable, as sizeof() resolves to a size_t

File: src/basic/linux/if.h [hunk 2]
```diff
@@ -1,4 +1,4 @@
-/* SPDX-License-Identifier: GPL-2.0+ WITH Linux-syscall-note */
+/* SPDX-License-Identifier: GPL-2.0-or-later WITH Linux-syscall-note */
```
Comment (poettering) [inline src/basic/linux/if.h]: these files are import from linux upstream, the spdx marker will be reset again when they are updated from upstream again

Comment (keszybz) [reply src/basic/linux/if.h]: Yeah, I was thinking that we might want to just ignore that part of the diff when applying. I wanted to have consistent spdx markers to make it easier to evaluate the whole tree using a script. But on second thought, updates would indeed be awkward. I'll drop that commit.

**Comment by keszybz** [comment]
I dropped the commit that touched kernel headers and added another one that touches our po files ;)
I also found a few more scripts in other directories that I missed in the first rounds.
Since it's all in the same vein, I'm taking the liberty to upgrade the label.

**Comment by yuwata** [comment]
LGTM.

**Comment by keszybz** [comment]
bionic-amd64 and -i386 failed in TEST-50-DISSECT, probably unrelated.

**Pull request merged**